modules/ecs-service/README.md

```diff
@@ -1,23 +1,204 @@
+
+Keep in mind that ECS service autoscaling is a powerful feature that automatically adjusts your service's capacity based on real-time performance metrics, ensuring optimal resource utilization and application responsiveness.
+
+#### ECS Service Connect
+
+ECS service connect enables easy communication between tasks. Set the `enable_service_connect` variable to `true` to enable this feature, available for ECS service types.
+
+Service connect replaces the ECS service discovery feature and creates a private DNS namespace for ECS services to share. It allows services to communicate using the `service-name:container-port` format as the hostname.
+
+- [ECS service connect](https://aws.amazon.com/blogs/aws/new-amazon-ecs-service-connect-enabling-easy-communication-between-microservices/)
+- [Service Connect Documentation](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect-concepts.html)
+
+#### ECS Service Rolling Deployment
+
+ECS service rolling deployment enables zero-downtime updates of your service. When an update is made, tasks are provisioned, and once they are registered as HEALTHY, old tasks are removed. Deployment failures can trigger rollback if `enable_deployment_rollback` is set to `true`.
+
+Deployment failures are marked when tasks fail to reach a steady state, defined by the minimum threshold of 10 tasks. This ensures that deployments proceed smoothly.
+
+- [ECS Rolling Deployment Circuit Breaker](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/deployment-circuit-breaker.html)
+- [Speeding up ECS deployments](https://docs.aws.amazon.com/AmazonECS/latest/bestpracticesguide/deployment.html)
+
+### ECS Scheduled Task
+
+#### Cron Expressions
 
 [Cron expressions reference](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-cron-expressions.html)
 [Rate expression reference](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-rate-expressions.html)
-[Amazon EventBridge events](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events.html)
+
+#### Event Patterns
+
 [Custom event pattern reference](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events-structure.html)
-[ECS service connect](https://aws.amazon.com/blogs/aws/new-amazon-ecs-service-connect-enabling-easy-communication-between-microservices/)
-[Service Connect Documentation](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect-concepts.html)
-[Load testing ECS application](https://ecsworkshop.com/monitoring/container_insights/performloadtest/)
-[Speeding up ECS deployments](https://docs.aws.amazon.com/AmazonECS/latest/bestpracticesguide/deployment.html)
+[Amazon EventBridge events](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events.html)
```

Comment:
What are these used for?

Reply:
These are the links to the documentation for these sections. The scheduled task was not implemented in the develop/test/documentation PRs because of the complexity of the PRs. However, a decent bit of work went towards it already so I just have the documentation so when the story eventually gets worked on we still have these references.